### PR TITLE
Use new symbiflow

### PR DESCRIPTION
**Pull request by acomodi**
This PR changes the symbiflow package that has new files location, as well as a far more reduced size.

### CODE REVIEW

File: env.sh [hunk 1]
```diff
@@ -16,5 +16,7 @@ if [ -z "${SYMBIFLOW}" ]; then
 fi
 
 export PATH=${SYMBIFLOW}/bin:${PATH}
+export PYTHONPATH=${FPGA_TOOL_PERF_BASE_DIR}/third_party/prjxray:${PYTHONPATH}
```
Comment (mithro) [inline env.sh:19]: Why is the `PYTHONPATH` back?

Comment (acomodi) [reply env.sh:19]: I actually need to check whether prjxray can be added in the python requirements, if that is possible I will use that.

Comment (acomodi) [reply env.sh:19]: For now this should be like this unfortunately. We could create yet another python package to extract the various prjxray scripts and install that from the requirements. Currently, the pythonpath needs to be there. 

I should add a FIXME though and a GH issue.

Comment (mithro) [reply env.sh:19]: I'm always more happy for you to land chances with FIXME's if they have a GitHub issue number associated with them.

Comment (acomodi) [reply env.sh]: Actually I have fixed this in https://github.com/SymbiFlow/prjxray/pull/1316. When that gets merged this PR should go green without the PYTHONPATH fix

Comment (acomodi) [reply env.sh]: @mithro PYTHONPATH fixed, there is no more need for that anymore

**Pull request merged**